frontend/public/components/utils/alerts.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+
+export const ExpandableAlert: React.FC<CustomAlertProps> = ({alerts, type}) => {
+  const errorCount = alerts.length;
+  const [visible, setVisible] = React.useState(true);
+  const [expanded, setExpanded] = React.useState(errorCount > 1 ? false : true);
+
+  if (visible) {
+    return <div className={`alert alert-${type}`}>
```

Comment:
If we're adding a component, I feel like we should wrap the Patternfly component and just change the message value. (Or be a child of a Patternfly alert component.) We're more or less reimplementing the Patternfly alert here.